lib/lambda-cfn.js

```diff
@@ -349,6 +340,9 @@ function lambda(options) {
       "Description": {
         "Ref": "AWS::StackName"
       },
+      "Environment": {
+        "Variables": {}
```

Comment:
@pauljlucas not sure if there's ever a case where there won't be any environment variables.  If there is, will this default be valid, or do you need to `delete` the property if no environment variables exist for the function?